apps/picsa-tools/budget-tool/src/app/components/budget-summary/budget-summary.component.scss

```diff
@@ -0,0 +1,29 @@
+.summary-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  margin-bottom: 1.2rem;
+
+  h2 {
+    margin: 0 0 0.5rem 0;
+    padding: 0;
+    font-size: 1rem;
+    color: #000;
+  }
+
+  .summary-list {
+    display: flex;
+    gap: 1rem;
+    flex-wrap: wrap;
+    justify-content: flex-start;
+    align-items: flex-start;
+
+    p {
+      margin: 0;
+      padding: 0;
+      font-size: 1rem;
```

Comment:
nit(non-blocking)
likely here the `font-size` and `color` styles are the same as inherited and not required